internal/golden/ai-sdk.ts

```diff
@@ -180,59 +179,31 @@ async function testDocumentInput() {
       );
 
       for (const [provider, model] of [
-        ["openai", openai("gpt-4o")],
+        ["openai", openai("gpt-5-mini")],
         ["anthropic", anthropic("claude-3-5-sonnet-20241022")],
       ] as const) {
         console.log(`${provider.charAt(0).toUpperCase() + provider.slice(1)}:`);
 
-        // Note: PDF file support in AI SDK appears to be limited
-        // OpenAI's AI SDK provider doesn't seem to process PDFs properly even with data URLs
-        // Anthropic has general issues with the AI SDK provider (not PDF-specific)
-        // This test demonstrates passing the file data correctly per AI SDK's FilePart interface
-        const messages =
-          provider === "openai"
-            ? [
+        const result = await generateText({
+          // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+          model: model as LanguageModel,
+          messages: [
+            {
+              role: "user" as const,
+              content: [
                 {
-                  role: "user" as const,
-                  content: [
-                    {
-                      type: "file" as const,
-                      file: {
-                        file_data: `data:application/pdf;base64,${base64Pdf}`,
-                        filename: "test-document.pdf",
-                      },
-                    },
-                    {
-                      type: "text" as const,
-                      text: "What is in this document?",
-                    },
-                  ],
+                  type: "file" as const,
```

Comment:
a lot of pain here, but turns out we were providing the wrong inputs here and in image tests